experiment/kind-detect-local-e2e.sh

```diff
@@ -54,12 +54,17 @@ signal_handler() {
 }
 trap signal_handler INT TERM
 
-# build kubernetes / node image, e2e binaries
+# build kubernetes / node image, e2e binaries and ginkgo
 build() {
   # build the node image w/ kubernetes
   kind build node-image -v 1
+  # Ginkgo v1 is used by Kubernetes 1.24 and earlier, fallback if v2 is not available.
+  GINKGO_SRC_DIR="vendor/github.com/onsi/ginkgo/v2/ginkgo"
```

Comment:
Let's drop the "vendor/" prefix. It is not needed right now and support for it seems to go away in https://github.com/thockin/kubernetes/tree/go-workspaces.